webgpu.h

```diff
@@ -164,6 +164,13 @@ typedef enum WGPUCompareFunction {
     WGPUCompareFunction_Force32 = 0x7FFFFFFF
 } WGPUCompareFunction;
 
+typedef enum WGPUCompilationMessageType {
+    WGPUCompilationMessageType_Error = 0x00000000,
+    WGPUCompilationMessageType_Warning = 0x00000001,
+    WGPUCompilationMessageType_Info = 0x00000002,
```

Comment:
Which option should be 0?

Reply:
Probably doesn't matter that much given it is only data passed from the implementation to the application.

Reply:
does it matter for this enum?

Reply:
agree it probably doesn't matter.